release.yaml

```diff
@@ -2,6 +2,8 @@ neuvector:
 - 100.0.0+up2.2.0-b2
 neuvector-crd:
 - 100.0.0+up2.2.0-b2
+prometheus-federator:
+- 0.1.0+up0.0.1
```

Comment:
Do we not want to follow the 100.x.x in this case?

Reply:
This falls under a chart that we own ourselves, so we don't need to add 100.x.x to it; examples of other charts like this are Backup Restore Operator and CIS Benchmark, which are both 1.x.x in 2.5, 2.x.x in 2.6.

The reason why we start at 0.x.x instead of 1.x.x is because CIS / Backup needed to differentiate from the system chart version, which was 0.x.x; this is not the case for Prometheus Federator since there is no system chart version of it.

The reason why we do 0.1.x instead of 0.0.x is because 0.0.x marks experimental versions released in the Prometheus Federator repository **only** that are not limited to Rancher 2.6.5+; it's deployable in earlier 2.6 versions of Rancher but would require a user to manually add a chart repository to do so. 

For Rancher, the first version we are officially supporting is 0.1.x+, which is scoped to 2.6.5+.